ext/standard/php_image.h

```diff
@@ -54,4 +55,6 @@ PHPAPI int php_getimagetype(php_stream *stream, const char *input, char *filetyp
 
 PHPAPI char * php_image_type_to_mime_type(int image_type);
 
+PHPAPI int php_is_image_avif(php_stream * stream);
```

Comment:
```suggestion
PHPAPI bool php_is_image_avif(php_stream *stream);
```
(Same for the definition.)

Reply:
thanks! When I did this, I didn't realize we had a convenient `bool` type here.